_posts/2025-04-28-sparse-autodiff.md

```diff
@@ -481,16 +491,18 @@ This idea is visualized in Figure 13.
     Figure 13: Propagating an index set through a linear map to obtain a sparsity pattern.  
 </div>
 
-### Alternative evaluation
+### Operator overloading
 
 Instead of going into implementation details,
-we want to provide some intuition on the second key ingredient of our forward-mode sparsity detection: 
-**alternative function evaluation**.
+we want to provide some intuition on a typical implementation technique for sparsity detection: 
+**operator overloading**.
```

Comment:
No, I'll explain this as alternative function evaluation (of which operator overloading would we one of many possible implementations).

Reply:
I reverted the change but I disagree with your chosen terminology. The term I have seen in the literature is "non-standard interpretation". I had never seen "alternative function evaluation" until today.

Reply:
Sure, I’ll go with that. I should have clarified that pattern detection is still WIP and not yet ready for iteration. 👍